docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -421,6 +423,192 @@ When I execute HTTP $httpMethod request for resource with URL `$url`
 When I execute HTTP GET request for resource with URL `https://example.com`
 ----
 
+=== Execute HTTP request with relative URL
+
+Executes the HTTP request for a resource identified by the URL relative to the base URL specified by the `rest-api.http.endpoint` property.
+
```

Comment:
do not put each sentence as a separate paragraph

Reply:
the sentences are completely different by their meaning, so why not to separate them logically into paragraphs?